user_view.html

```diff
@@ -53,7 +53,9 @@ <h2> Your Recurring Donations </h2>
 <li>{{ profile.amt }} a {{ profile.inverse_period_display|lower }}, started on {{ profile.created_at|date:"m/d/Y" }}. 
 You've given a total of {{ profile.payment_total_amt }}. 
 {% if profile.is_active %}
-  {% if profile.order.payment_method != "paypal" %}
+  {% if 'ActBlue' in profile.account %}
+    <a href="https://secure.actblue.com/signin">Update your credit card or cancel at any time - via ActBlue.</a>
```

Comment:
Could users realistically use both ActBlue and/or Action Kit or Paypal?

Reply:
this is for each recurring donation, see: https://github.com/350org/ak_intl_v3/pull/405/files/613995a648ae948a936fc1ec12d3e40e3f36e5a9#diff-fc37e8d1df46f7163b0d93668ed8281f630a56dbfbca6eb549df064f7c99766bL52. so a user could use actblue for one recurring donation, and paypal for another, but those would be rendered differently.